src/github-release.ts

```diff
@@ -129,7 +129,18 @@ export default class GitHubRelease {
         args.baseUrl = this.githubApi;
       }
 
-      this.logger.verbose.info('Initializing GitHub API with:\n', args);
+      // So that --verbose can be used on public CIs
+      const tokenlessArgs = {
+        ...args,
+        token: args.token
+          ? `[Token starting with ${args.token.substring(0, 4)}]`
+          : undefined
+      };
+
+      this.logger.verbose.info(
+        'Initializing GitHub API with:\n',
+        tokenlessArgs
+      );
```

Comment:
y'all should move to 120 chars BTW :P

Reply:
For prettier? I'll take a look at what that looks like tonight. The default is nice for side by side editing on VSCode.